test/nondefaulte2e/suite_test.go

```diff
@@ -21,11 +21,14 @@ import (
 	"os"
 	"path/filepath"
 	"strings"
+	"testing"
 	"time"
 
 	// Import all Kubernetes client auth plugins (e.g. Azure, GCP, OIDC, etc.)
 	// to ensure that exec-entrypoint and run can make use of them.
+	argov1alpha1api "github.com/argoproj-labs/argocd-operator/api/v1alpha1"
 	argoapi "github.com/argoproj-labs/argocd-operator/api/v1beta1"
```

Comment:
Can we remove the duplicate imports?

This 
```suggestion
```

and this
https://github.com/redhat-developer/gitops-operator/blob/1f42107fc31c2ae810872bca49746036fa660642/test/nondefaulte2e/suite_test.go#L97

Reply:
Done